interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -61,5 +124,83 @@ paths:
           description: The access token was decoded successfully but did not include a scope appropriate to this endpoint.
         '404':
           description: The USS automated testing interface is not activated.
-      summary: Retrieve the status of the USS automated testing interface
-      description: Get the status of the USS automated testing interface
+      summary: Status of the USS automated testing interface
+      description: Get the status of the USS automated testing interface.
+
+
+  /v1/geozones/{geozones_id}:
+    parameters:
+      - name: geozones_id
+        in: path
+        required: true
+        description: A UUID string identifying the injected geozones dataset.
+        schema:
+          $ref: '#/components/schemas/UUIDv4Format'
+
+    put:
+      security:
+        - Authority:
+            - utm.inject_test_data
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InjectGeozonesRequest'
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InjectGeozonesResponse'
+          description: >-
+            Request has been correctly handled and the geozones dataset will be imported and activated by the USS.
+        '401':
+          description: Bearer access token was not provided in Authorization header, token could not be decoded, or token was invalid.
+        '403':
+          description: The access token was decoded successfully but did not include a scope appropriate to this endpoint.
+      operationId: PutInjectGeozones
+      summary: Activate a Geozones dataset
+      description: Instructs the USS to import and activate a Geozones dataset.
+
+    get:
+      security:
+        - Authority:
+            - utm.inject_test_data
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InjectGeozonesResponse'
+          description: >-
+            The Geo-Awareness dataset is handled by the USS and its status was retrieved successfully.
+        '401':
+          description: Bearer access token was not provided in Authorization header, token could not be decoded, or token was invalid.
+        '403':
+          description: The access token was decoded successfully but did not include a scope appropriate to this endpoint.
+        '404':
+          description: The Geozones dataset has been successfully deleted or didn't exist.
+      operationId: GetInjectGeozones
```

Comment:
Let's either simplify this to `GetGeozone` (singular, since we're only getting one of them with this operation) or else change the resource path name to `injected_geozones` rather than just `geozones` (`/v1/injected_geozones/{injected_geozone_id}` -> GetInjectedGeozone).  RESTful endpoints should be based on a common resource name ("geozone" or "injected geozone") which has simple actions associated with it.

Reply:
You are actually highlighting a doubt I had when working on the PR because the reference we inject points to a json file which contains a set a of geozones. So it is not entirely correct to use geozone as a resource since the entity is actually a set of geozone. What do you think would be the best approach?

Reply:
Yeah, that makes sense.  Maybe the resource name should be `geozone_source` or `geozone_set` or `geozone_collection`?  (so, e.g., GET /v1/geozone_sources/{source_id} -> GetGeozoneSource)

Reply:
Changed to `/v1/geozone_sources/{geozone_source_id}` and `GetGeozoneSource`